examples/access_control_function/get_access_control_function.tf

```diff
@@ -0,0 +1,13 @@
+output "existing_acess_control_function" {
+    value = data.couchbase-capella_access_control_function.existing_acess_control_function
+}
+
+data "couchbase-capella_access_control_function." "existing_acess_control_function" {
+  organization_id         = var.access_control_function.organization_id
+  project_id              = var.access_control_function.project_id
+  cluster_id              = var.access_control_function.cluster_id
+  app_service_id          = var.access_control_function.app_service_id
+  app_endpoint_name       = var.access_control_function.app_endpoint_name
+  scope                   = var.access_control_function.scope
+  collection              = var.access_control_function.collection
```

Comment:
app_endpoint_name  does not match schema, however I am using it as app_endpoint_name on activation status, so would be viable to change the importIds value or wait for that PR to go in

Reply:
addressed in https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/390/commits/d0f107e430056edfa02af40910f1a41f9f7efdc1